src/helpers.js

```diff
@@ -56,3 +56,11 @@ export function idsToIgnoredRules(ruleIds = []) {
     return ids
   }, {})
 }
+
+export function validatePoint(textEditor, line, col) {
+  const buffer = textEditor.getBuffer()
+  // Clip the given point to a valid one, and check if it equals the original
+  if (!buffer.clipPosition([line, col]).isEqual([line, col])) {
+    throw new Error(`${line}:${col} isn't a valid point!`)
```

Comment:
I'm not convinced that throwing an error is the right thing to do here.  We could be more graceful and just use the clipped position.  I suppose the positive here is that we might be able to catch ESLint errors, whereas without throwing an error people will just live with the possibly slightly-inaccurate location marking.  Pros and cons either way, but personally I think I lean towards not throwing an error.  Curious to hear your thoughts.